### ISSUE TITLE
AC_PROG_LIBTOOL is deprecated

### ISSUE THREAD

**Issue by bkmgit** [pantoniou/libfyaml#90]
replace with LT_INIT

**Comment by bkmgit** [comment]
Other obsolete macros
```
AC_CONFIG_HEADER
AC_PROG_CC_C99
AC_LIBLTDL_INSTALLABLE
AC_LIBTOOL_DLOPEN
AC_LIBTOOL_WIN32_DLL
AC_HEADER_STDC
AC_TYPE_SIGNAL
```
Tested with autoconf 2.71 and automake 1.16.5

**Comment by pantoniou** [comment]
Fixed by 01c441ff881ffcbfba32b603e5033344b9fc3a5b
